bigflow/build/dev.py

```diff
@@ -0,0 +1,98 @@
+"""Build-related developer tools and helpers.
+
+Intented to be used by "CLI" and in "development environement".
+"""
+
+import sys
+import logging
+import tempfile
+import pickle
+import typing
+import os
+import os.path
+
+from typing import Optional, List, Union
+from pathlib import Path
+
+import bigflow.commons as bfc
+import bigflow.build.reflect
+
+
+logger = logging.getLogger(__name__)
+
+# Used by 'dist.py' to dump `bigflow.setup(...)` arguments.
```

Comment:
This comment is redundant

Reply:
?

Reply:
This comment does not provide any information.

Reply:
Comment has been [changed].(https://github.com/allegro/bigflow/pull/164/commits/19a7807cfdc45e26923b8ae873d28c33f70a2ff4)